test/test_sendfile.rs

```diff
@@ -28,3 +35,42 @@ fn test_sendfile() {
     close(rd).unwrap();
     close(wr).unwrap();
 }
+
+#[cfg(target_os = "freebsd")]
+#[test]
+fn test_sendfile_freebsd() {
+    // Declare the content
+    let header_strings = vec!["HTTP/1.1 200 OK\n", "Content-Type: text/plain\n", "\n"];
+    let body = "Xabcdef123456";
+    let body_offset = 1;
+    let trailer_strings = vec!["\n", "Served by Make Believe\n"];
+
+    // Write the body to a file
+    let mut tmp = tempfile().unwrap();
+    tmp.write_all(body.as_bytes()).unwrap();
+
+    // Prepare headers and trailers for sendfile
+    let headers: Vec<IoVec<&[u8]>> = header_strings.iter().map(|s| IoVec::from_slice(s.as_bytes())).collect();
+    let trailers: Vec<IoVec<&[u8]>> = trailer_strings.iter().map(|s| IoVec::from_slice(s.as_bytes())).collect();
+
+    let hdtr = SfHdTr::new(headers.as_slice(), trailers.as_slice());
+
+    // Prepare socket pair
+    let (mut rd, wr) = UnixStream::pair().unwrap();
+
+    // Call the test method
+    let (res, bytes_written) = sendfile(tmp.as_raw_fd(), wr.as_raw_fd(), body_offset as off_t, 0, Some(&hdtr), SfFlags::empty(), 0);
```

Comment:
Wrap to 80 columns, please.

Reply:
Should I just run `rustfmt` on the files I've touched?